test/performance/Benchmarks.md

```diff
@@ -0,0 +1,166 @@
+# Benchmarks
+
+Knative performance benchmarks are tests geared towards producing useful
+performance metrics of the knative system. All the raw metrics are stored in
+[mako](https://github.com/google/mako)
+
+## Writing new benchmarks
+
+For creating new benchmarks, follow the steps:
+
+1. Create a new directory under `./test/performance/`.
+2. Create two benchmarks using in the bechmark directory using
+   [mako](https://github.com/google/mako/blob/github-push-test-1/docs/GUIDE.md#preparing-your-benchmark)
+   as mentioned in [benchmark configs](#benchmark-configs).
+3. Create a `kodata` directory and add the [four symlinks](#Benchmark-Symlinks).
+4. Write a go program that runs the test and stores the result in
+   [mako](##Writing-to-mako)
+5. Create a [setup.yaml](https://github.com/knative/serving/blob/master/test/performance/dataplane-probe/dataplane-probe-setup.yaml)
+   that lists all the K8S and Knative objects needed to run the test.
+6. Create a [cron.yaml](https://github.com/knative/serving/blob/master/test/performance/dataplane-probe/dataplane-probe.yaml)
+   that defines how to run and capture metrics as mentioned in
+   [benchmark cronjobs](#Benchmark-cronjobs).
+7. Ask one of the [admins](#Admins) to run the `create_cluster_benchmark.sh`
```

Comment:
weird, the admins link doesn't show up in the MD view.

Reply:
I can see them in the md view as well.

Reply:
I meant Github's markdown diff viewer, which is usually pretty good.  I didn't try loading the page directly.